spec/operators/mergeMap-spec.ts

```diff
@@ -283,92 +229,6 @@ describe('Observable.prototype.mergeMap', () => {
     expectSubscriptions(e1.subscriptions).toBe(e1subs);
   });
 
-  it('should mergeMap to many cold Observable, with parameter concurrency=1', () => {
-    const values = {i: 'foo', j: 'bar', k: 'baz', l: 'qux'};
-    const e1 =     hot('-a-------b-------c---|                                        ');
-    const e1subs =     '^                                                            !';
-    const inner =  cold('----i---j---k---l---|                                        ', values);
-    const innersubs = [' ^                   !                                        ',
-                     '                     ^                   !                    ',
-                     '                                         ^                   !'];
-    const expected =   '-----i---j---k---l-------i---j---k---l-------i---j---k---l---|';
-
-    function project() { return inner; }
-    function resultSelector(oV: string, iV: string, oI: number, iI: number) { return iV; }
-    const result = e1.mergeMap(project, resultSelector, 1);
```

Comment:
It looks like these tests shouldn't have been removed, instead just removing the usage of resultSelector